tests/unit/amo/components/TestAddAddonToCollection.js

```diff
@@ -294,8 +294,10 @@ describe(__filename, () => {
       const promptOption = findOption({
         root, text: 'Select a collection…',
       });
+      expect(promptOption).toHaveProp('disabled', true);
 
-      // Select the prompt (first option) which doesn't do anything.
+      // Just to cover the error handling code, try selecting the prompt
```

Comment:
I think this comment is a bit oddly-worded and I don't exactly follow what it's documenting. It's being selected to make sure nothing happens, right? It's a bit vague, maybe this could be reworded... I think the original wording is actually a bit clearer.